src/ui/components/Rating/index.js

```diff
@@ -66,18 +71,35 @@ export default class Rating extends React.Component {
   }
 
   render() {
-    const { readOnly, size } = this.props;
+    const { i18n, rating, readOnly, size } = this.props;
     const cls = classNames('Rating', {
```

Comment:
Could this be `className` instead of `cls`? I find those short vars hard to read best of times but this one actually confused me a bit.